pkg/liquidity-mining/contracts/admin/GaugeAdder.sol

```diff
@@ -37,22 +37,24 @@ contract GaugeAdder is IGaugeAdder, SingletonAuthentication, ReentrancyGuard {
     // Mapping from mainnet BPT addresses to canonical liquidity gauge as listed on the GaugeController
     mapping(IERC20 => ILiquidityGauge) internal _poolGauge;
 
-    constructor(IGaugeController gaugeController, IGaugeAdder previousGaugeAdder)
-        SingletonAuthentication(gaugeController.admin().getVault())
-    {
+    constructor(
+        IGaugeController gaugeController,
+        IGaugeAdder previousGaugeAdder,
+        IAuthorizerAdaptorEntrypoint authorizerAdaptorEntrypoint
+    ) SingletonAuthentication(gaugeController.admin().getVault()) {
         _gaugeController = gaugeController;
-        _authorizerAdaptor = gaugeController.admin();
+        _authorizerAdaptorEntrypoint = authorizerAdaptorEntrypoint;
         _previousGaugeAdder = previousGaugeAdder;
 
         // Cache the BAL 80 WETH 20 BPT on this contract.
         _balWethBpt = gaugeController.token();
     }
 
     /**
-     * @notice Returns the address of the Authorizer adaptor contract.
+     * @notice Returns the address of the Authorizer adaptor entrypoint contract.
      */
-    function getAuthorizerAdaptor() external view override returns (IAuthorizerAdaptor) {
-        return _authorizerAdaptor;
+    function getAuthorizerAdaptorEntrypoint() external view override returns (IAuthorizerAdaptorEntrypoint) {
```

Comment:
I was in doubt yesterday when I did this fix: should we also add a getter for the adaptor?

I think it's fine as it is, since you could also get the adaptor from the entrypoint's getter, but I wanted to double check JIC.

Reply:
I also think it's fine as is - I don't think there's a need to have it externally (and could be confusing). Plus as you point out, there's another way to get it if necessary.